dev/com.ibm.ws.kernel.boot.nested/src/com/ibm/ws/kernel/launch/internal/FrameworkManager.java

```diff
@@ -232,39 +232,40 @@ public void launchFramework(final BootstrapConfig config, final LogProvider logP
             String j2secNoRethrow = config.get(BootstrapConstants.JAVA_2_SECURITY_NORETHROW);
 
             if (j2secManager) {
-
-                if (j2secNoRethrow == null || j2secNoRethrow.equals("false")) {
-                    try {
-                        AccessController.doPrivileged(new java.security.PrivilegedExceptionAction<Void>() {
-                            @Override
-                            public Void run() throws Exception {
-                                System.setSecurityManager(new SecurityManager());
-                                return null;
-                            }
-                        });
-                    } catch (Exception ex) {
-
-                        Tr.error(tc, "error.set.securitymanager", ex.getMessage());
-                    }
+                // OLGH#20289 -- Java 2 Security Manager is no longer supported with Java 18+
```

Comment:
I'm not sure what that would save?  We don't want to emit this message just because Java 18 is being used, we'd want it to be emitted if "websphere.java.security" is set in bootstrapping.properties and Java 18 is being used.

Reply:
I wasn't suggesting changing the outcome.  Was really just wanting to isolate the logic out to a new method because I think it would be more clear.  Once we support only Java versions where no security manager can be set then we can just delete the whole method.

But we don't have to do that now.  I just like to avoid adding new nestings of if statements if not necessary.  It would have looked something like this:

```java
    private void enableJ2secManager() {
        if (javaVersion() >= 18) {
            Tr.error(tc, "error.set.securitymanager.jdk18", javaVersion());
            return;
        }
        // do all the security manager stuff ...
    }
```

Just something nice to have, not required for this PR.